units/en/unit1/quiz.mdx

```diff
@@ -0,0 +1,168 @@
+- Our Agent will move to the right
+- The Environment goes to a ______
+- The Environment gives a ______ to the Agent
+
+
+<Question
+	choices={[
+		{
+			text: "an action a0, action a0, state s0, state s1, reward r1",
+			explain: "At every step: Our Agent receives **state s0** from the environment. Based on that **state s0** the Agent takes an **action a0**. Our Agent will move to the right. The Environment goes to a **new state s1**. The Environment gives **a reward r1** to the Agent."
+		},
+		{
+			text: "state s0, state s0, action a0, new state s1, reward r1",
+			explain: "",
+      correct: true
+		},
+		{
+			text: "a state s0, state s0, action a0, state s1, action a1",
+      explain: "At every step: Our Agent receives **state s0** from the environment. Based on that **state s0** the Agent takes an **action a0**. Our Agent will move to the right. The Environment goes to a **new state s1**. The Environment gives **a reward r1** to the Agent."
+		}
+	]}
+/>
+
+### Q3: What's the difference between a state and an observation?
+
+<Question
+	choices={[
+		{
+			text: "The state is a complete description of the state of the world (there is no hidden information)",
+			explain: "",
+      correct: true
+		},
+    {
+			text: "The state is a partial description of the state",
+			explain: ""
+		},
+    {
+      text: "The observation is a complete description of the state of the world (there is no hidden information)",
+      explain: ""
+    },
+    {
+      text: "The observation is a partial description of the state",
+      explain: "",
+      correct: true
+    },
+    {
+      text: "We receive a state when we play with chess environment",
+      explain: "Since we have access to the whole checkboard information.",
+      correct: true
+    },
+    {
+      text: "We receive an observation when we play with chess environment",
+      explain: "Since we have access to the whole checkboard information."
+    },
+    {
+      text: "We receive a state when we play with Super Mario Bros",
+      explain: "We only see a part of the level close to the player, so we receive an observation."
+    },
+    {
+      text: "We receive an observation when we play with chess environment",
+      explain: "We only see a part of the level close to the player.",
+      correct: true
+    }
+	]}
+/>
+
+### Q4: A task is an instance of a Reinforcement Learning problem. What are the two types of tasks?
+
+<Question
+	choices={[
+		{
+			text: "Episodic",
+			explain: "In Episodic task, we have a starting point and an ending point (a terminal state). This creates an episode: a list of States, Actions, Rewards, and new States. For instance, think about Super Mario Bros: an episode begin at the launch of a new Mario Level and ending when you’re killed or you reached the end of the level.",
+      correct: true
+		},
+    {
+			text: "Recursive",
+			explain: ""
+		},
+    {
+			text: "Adversarial",
+			explain: ""
+		},
+    {
+      text: "Continuing",
```

Comment:
To rename to continuous

Reply:
Continuing is the official Sutton definition initially we put continuous and a student pointed the resource (Sutton book) so I've changed the term.